service/src/main/java/datawave/microservice/audit/auditors/accumulo/config/AccumuloAuditProperties.java

```diff
@@ -65,4 +87,42 @@ public void setPassword(String password) {
             this.password = password;
         }
     }
+    
+    public static class Health {
+        private Long hungTimeout = 30L;
```

Comment:
I thought we were shooting for 5 miinutes.

Reply:
that is correct.  these were my test values.  updating...